modules/logging-release-notes-6-2-4.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * release_nates/logging-release-notes-6.2.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-6-2-4_{context}"]
+= Logging 6.2.4 Release Notes
+
+//This release includes link:https://access.redhat.com/errata/RHBA-2025:8138[RHBA-2025:8138].
+// To be updated after the release
+
+[id="logging-release-notes-6-2-4-bug-fixes_{context}"]
+== Bug Fixes
+
+* Before this update, using the `prune` filter to remove Kubernetes metadata fields such as `.kubernetes.namespace_name` caused a `remapping` error during Syslog output generation. This prevented the `appname` and `tag` fields from being correctly initialized and could prevent forwarding of logs. With this update, improved error handling initializes these fields with an empty string if the source metadata is missing, resolving the issue and ensuring successful log processing. (link:https://issues.redhat.com/browse/LOG-7267[LOG-7267])
+
+*  Before this update, a `ClusterLogForwarder` CR that was configured for a `LokiStack` output with the OTEL data model incorrectly passed validation without the `tech preview` annotation. With this update, a `ClusterLogForwarder` CR that is configured for a `LokiStack` output with the OTEL data model correctly fails validation unless the `tech preview` annotation is included. (https://issues.redhat.com/browse/LOG-7280[LOG-7280])
```

Comment:
Can we expand OTEL as per the appropriate term? Based on a quick search, I think it is OpenTelemetry as per the Observability docs.